web/src/routes/index.ts

```diff
@@ -1,5 +1,6 @@
 import Map from 'components/Map'
 import Upload from 'components/Upload'
+import UploadConfirmation from 'components/Upload_Confirmation'
```

Comment:
Could we swap out the underscore in the file name to make it `UploadConfirmation`? The `ControlPanel` component within the `Map` container set a naming precedent for components as uppercase, no snakes.